### PR TITLE
added market root command

### PR DESCRIPTION
**Pull request by cis-suyash**
1)Added marketRootCommand and marketListCommand.

### CODE REVIEW

**Comment by cis-suyash** [comment]
Currently I have not fixed the black-box test cases for the create market and list market. once the dataRootCommand PR will be merge then I will update the black-box test cases for the market.

File: test/black-box/MarketCreateCommand.test.ts [hunk 1]
```diff
@@ -7,7 +7,7 @@ describe('MarketCreateCommand', () => {
 
     const testUtil = new BlackBoxTestUtil();
     const marketService = null;
-    const method =  new MarketCreateCommand(marketService, Logger).name;
+    const method =  'market';
```
Comment (xludx) [inline test/black-box/MarketCreateCommand.test.ts]: use Commands.ASDF

File: test/black-box/MarketListCommand.test.ts [hunk 2]
```diff
@@ -0,0 +1,29 @@
...
+import { BlackBoxTestUtil } from './lib/BlackBoxTestUtil';
+import { Logger } from '../../src/core/Logger';
+import { MarketListCommand } from '../../src/api/commands/market/MarketListCommand';
+
+describe('MarketListCommand', () => {
+
+    const testUtil = new BlackBoxTestUtil();
+    const marketService = null;
+    const method =  'market';
+
```
Comment (xludx) [inline test/black-box/MarketListCommand.test.ts]: use Commands

File: test/black-box/MarketListCommand.test.ts [hunk 3]
```diff
@@ -0,0 +1,29 @@
...
+    beforeAll(async () => {
+        await testUtil.cleanDb();
+    });
+
+    test('Should list all markets', async () => {
+        // generate market
+        // const listingItems = await testUtil.generateData('market', 3);
```
Comment (xludx) [inline test/black-box/MarketListCommand.test.ts]: test

**Pull request merged**